src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double base1;
+    private double base2;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double base1, double base2, double height) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+    }
+
+
+    @Override
+    public double area() {
+        return 0.5 * (base1 + base2) * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Isosceles Trapezoid, Color: " + color + ", Base1: " + base1 + ", Base2: " + base2 + ", Height: " + height + ", Area: " + area());
```

Comment:
When overriding the `draw` method, it's better to use the `toString` method within the `System.out.println` call to print the object's description. This separates the concerns of object description and drawing.